### ISSUE TITLE
Is it possible to release the training recipe?

### ISSUE THREAD

**Issue by shuaitang** [Cadene/skip-thoughts.torch#1]
Thank you so much for the skip-thought vector in Torch, which will potentially help me a lot. I was wondering if it is possible to release the training recipe?

**Comment by Cadene** [comment]
Glad to help you :)

What do you mean by training recipe ? I did not train any models. I converted them from the original pretrained models in theano : https://github.com/ryankiros/skip-thoughts

Note that the outputs of my pretrained models in torch are the same than those from theano.

However I did not even test their accuracy on the experimental tasks from the orignal paper. Nevertheless I am using this lib for my personal research projects and it works very well.

**Comment by shuaitang** [comment]
Yep, I was wondering the test accuracy on the 8 tasks mentioned in the skip-thought paper.

If it works pretty good, then I think I'm gonna use it too.

Thank you!
